.env

```diff
@@ -1,13 +1,14 @@
 # React Environment Variables
 # https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables#expanding-environment-variables-in-env
-REACT_APP_API_URL=https://kbss.felk.cvut.cz/sgov-server
-REACT_APP_OIDC_AUTHORITY=https://kbss.felk.cvut.cz:10808/auth/realms/kodi-dev
-REACT_APP_OIDC_CLIENT_ID=kodi-mission-control-dev
-REACT_APP_BUG_TRACKER_URL=https://github.com/opendata-mvcr/sgov-assembly-line/issues/new?labels=bug&template=po-adavek-na-opravu.md&title=
-REACT_APP_FEATURE_TRACKER_URL=https://github.com/opendata-mvcr/sgov-assembly-line/issues/new?labels=enhancement&template=po-adavek-na-novou-funkcionalitu.md&title=
+
+REACT_APP_ENV=production
+REACT_APP_ID=kodi-mission-control-dev
+REACT_APP_URL=https://localhost:8080
+REACT_APP_COMPONENTS=ewogICJzZ292LXNlcnZpY2UiOiB7CiAgICAibmFtZSI6ICJTR29WIFNlcnZpY2UiLAogICAgInVybCI6ICJodHRwczovL2tic3MuZmVsay5jdnV0LmN6L3Nnb3Ytc2VydmVyIiwKICAgICJtZXRhIjoge30KICB9LAogICJkYiI6IHsKICAgICJuYW1lIjogIkEgZ3JhcGggZGF0YWJhc2UiLAogICAgInVybCI6ICJodHRwczovL2V4YW1wbGUuY29tL3NwYXJxbCIsCiAgICAibWV0YSI6IHt9CiAgfSwKICAiYXV0aCI6IHsKICAgICJuYW1lIjogIk9JREMgQXV0aGVudGljYXRpb24gU2VydmljZSIsCiAgICAidXJsIjogImh0dHBzOi8va2Jzcy5mZWxrLmN2dXQuY3o6MTA4MDgvYXV0aC9yZWFsbXMva29kaS1kZXYiLAogICAgIm1ldGEiOiB7fQogIH0sCiAgIm9udG9ncmFwaGVyIjogewogICAgIm5hbWUiOiAib250b0dyYXBoZXIiLAogICAgInVybCI6ICJodHRwczovL3Nsb3Zuw61rLmdvdi5jei9tb2RlbHVqZW1lL3YtbsOhc3Ryb2ppL29udG9ncmFwaGVyIiwKICAgICJtZXRhIjogewogICAgICAid29ya3NwYWNlUGF0aCI6ICI/d29ya3NwYWNlPSVXT1JLU1BBQ0VfSVJJJSIKICAgIH0KICB9LAogICJ0ZXJtaXQtYmFja2VuZCI6IHsKICAgICJuYW1lIjogIlRlcm1JVCBiYWNrZW5kIiwKICAgICJ1cmwiOiAiaHR0cHM6Ly9leGFtcGxlLmNvbS90ZXJtaXQtYmFja2VuZCIsCiAgICAibWV0YSI6IHt9CiAgfSwKICAidGVybWl0IjogewogICAgIm5hbWUiOiAiVGVybUlUIiwKICAgICJ1cmwiOiAiaHR0cHM6Ly9zbG92bsOtay5nb3YuY3ovbW9kZWx1amVtZS92LW7DoXN0cm9qaS90ZXJtaXQiLAogICAgIm1ldGEiOiB7CiAgICAgICJ3b3Jrc3BhY2VQYXRoIjogIiN2b2NhYnVsYXJpZXM/d29ya3NwYWNlPSVXT1JLU1BBQ0VfSVJJJSIKICAgIH0KICB9LAogICJtaXNzaW9uLWNvbnRyb2wiOiB7CiAgICAibmFtZSI6ICJNaXNzaW9uIENvbnRyb2wiLAogICAgInVybCI6ICJodHRwczovL2V4YW1wbGUuY29tL21pc3Npb24tY29udHJvbCIsCiAgICAibWV0YSI6IHt9CiAgfSwKICAiaXNzdWUtdHJhY2tlciI6IHsKICAgICJuYW1lIjogIklzc3VlIFRyYWNrZXIiLAogICAgInVybCI6ICJodHRwczovL2dpdGh1Yi5jb20vb3BlbmRhdGEtbXZjci9zZ292LWFzc2VtYmx5LWxpbmUvaXNzdWVzIiwKICAgICJtZXRhIjogewogICAgICAibmV3QnVnIjogImh0dHBzOi8vZ2l0aHViLmNvbS9vcGVuZGF0YS1tdmNyL3Nnb3YtYXNzZW1ibHktbGluZS9pc3N1ZXMvbmV3P2xhYmVscz1idWcmdGVtcGxhdGU9cG8tYWRhdmVrLW5hLW9wcmF2dS5tZCZ0aXRsZT0iLAogICAgICAibmV3RmVhdHVyZSI6ICJodHRwczovL2dpdGh1Yi5jb20vb3BlbmRhdGEtbXZjci9zZ292LWFzc2VtYmx5LWxpbmUvaXNzdWVzL25ldz9sYWJlbHM9ZW5oYW5jZW1lbnQmdGVtcGxhdGU9cG8tYWRhdmVrLW5hLW5vdm91LWZ1bmtjaW9uYWxpdHUubWQmdGl0bGU9IgogICAgfQogIH0KfQo=
```

Comment:
This is a base64 encoded sample components configuration; the original is in components.json file.